packages/components/src/components/pin-code/pin-code.examples.md

```diff
@@ -0,0 +1,140 @@
+# Pin Code
+
+The `p-pin-code` component ...
+
+<TableOfContents></TableOfContents>
+
+## Basic
+
+<Playground :markup="hideLabelMarkup" :config="config">
+  <SelectOptions v-model="hideLabel" :values="hideLabels" name="hideLabel"></SelectOptions>
+</Playground>
+
+## With description text
+
+A description text can be added to explain the meaning of the `p-pin-code` component. It's meant to be a textual
+enhancement of the label text and is technically connected with the `hide-label` property.
+
+<Playground :markup="withDescriptionText" :config="config"></Playground>
+
+## Length
+
+<Playground :markup="lengthMarkup" :config="config">
+  <SelectOptions v-model="length" :values="lengths"></SelectOptions>
+</Playground>
+
+## Type
+
+<Playground :markup="typeMarkup" :config="config">
+  <SelectOptions v-model="type" :values="types"></SelectOptions>
+</Playground>
+
+## Validation states
+
+The `p-pin-code` component supports the visualisation of inline validation. The `message` is colored and visible/hidden
+depending on the defined `state`.
+
+<Playground :markup="stateMarkup" :config="config">
+  <SelectOptions v-model="state" :values="states" name="state"></SelectOptions>
+</Playground>
+
+## Disabled
+
+<Playground :markup="disabledMarkup" :config="config"></Playground>
+
+### <A11yIcon></A11yIcon> Accessibility hints
+
+In general, you should **prevent** using the `disabled="true"` state. Disabled elements are not reachable (focusable)
+anymore and can be missed by screen reader users. They can be confusing for sighted users as well by not pointing out
+why these elements are disabled. A good practice when to use the disabled state is during **form submission** to prevent
+changes while this process is performed.
+
+## Loading
+
+## Required
+
+<Playground :markup="requiredMarkup" :config="config"></Playground>
+
+## Framework Implementation (within form)
+
+<Playground :frameworkMarkup="formExample" :config="{ ...config, withoutDemo: true }">
+<form @submit.prevent="onSubmit" >
+  <p-pin-code :theme="theme" label="Some Label" name="pin-code"></p-pin-code>
+  <p-button type="submit" style="margin: 1rem 0">Submit</p-button>
+</form>
+  <p-text :theme="theme">Last submitted data: {{ currentValueForm }}</p-text>
+</Playground>
+
+## Framework implementation (controlled)
```

Comment:
Not sure if controlled makes sense here since its not really controlled from outside